LibsUnix.sln

```diff
@@ -0,0 +1,364 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
suggestion:  this is really "libs netcore / cross plat".  should we name this LibsCore since it could also be useful for rapid development on Windows or Mac as well?